public/app/ui/components/search/results/search-results.html

```diff
@@ -25,7 +25,7 @@
                     <thead>
                     <tr>
                         <th>#</th>
-                        <!-- ko foreach: { data: fields, as: 'field' } -->
+                        <!-- ko foreach: { data: resultFields, as: 'field' } -->
```

Comment:
@mostro20 one of the refactors broke the tabular results view. I wasn't sure how to find the correct names for these variables, but after some digging I found `SearchResultsComponent.js` had a resultFields.